sms/message.go

```diff
@@ -96,14 +98,14 @@ func Read(c *messagebird.Client, id string) (*Message, error) {
 	return message, nil
 }
 
-// Cancel sending Scheduled Sms.
-func Delete(c *messagebird.Client, id string) (*Message, error) {
-	message := &Message{}
-	if err := c.Request(message, http.MethodDelete, path+"/"+id, nil); err != nil {
-		return nil, err
+// Delete Cancel sending Scheduled Sms.
+// Return true if have been successfully deleted.
+func Delete(c *messagebird.Client, id string) (bool, error) {
```

Comment:
Considering this will change the public API of the SDK, this should be considered a breaking change. I suppose in this scenario that's warranted. If you agree, can you please make a major version upgrade?

Reply:
It makes sense, I left only error in return and updated tests.
https://github.com/messagebird/go-rest-api/pull/119/commits/17648c3c4bec6ca8092427343bf162ae5ae40bc8

Reply:
Yep, updated